src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkUnix.cs

```diff
@@ -22,6 +22,9 @@ class MonoDroidSdkUnix : MonoDroidSdkBase
 		protected override string FindRuntime ()
 		{
 			string monoAndroidPath = Environment.GetEnvironmentVariable ("MONO_ANDROID_PATH");
+			if (monoAndroidPath == null)
+				// for Mono.Posix and Mono.Data.Sqlite builds in xamarin-android.
+				monoAndroidPath = Path.GetFullPath (Path.Combine (new Uri (GetType ().Assembly.CodeBase).LocalPath, "..", "..", "..", "..", ".."));
```

Comment:
My point was this feels like a hack. We could instead add the relative path to the "additionalSearchPaths" variable [1] which would be more in keeping of how this method works. 

[1] https://github.com/atsushieno/xamarin-android/blob/8db545653004741711f39d75fa598e1f5a88ace6/src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkUnix.cs#L40